src/main/java/org/broadinstitute/hellbender/engine/GATKTool.java

```diff
@@ -949,11 +942,11 @@ public VariantContextWriter createVCFWriter(final Path outPath) {
                     options.toArray(new Options[0]));
         }
 
-        return outputVariantIntervalFilteringMode== null ?
+        return getVariantFilteringOutputModeIfApplicable() == IntervalFilteringVcfWriter.Mode.ANYWHERE ?
```

Comment:
I'm not quite sure this rename helps that much, it just adds another layer of indirection.  It's then kind of confusing because tools below variantwalker might not be sure which to override.

Reply:
changed to `getVariantOutputFilteringMode` I don't really know what **IS** better here